### DISCUSSION TITLE
How to create custom placeholder?

### DISCUSSION THREAD

**Discussion by ghost** [TownyAdvanced/Towny#5857] [Q&A]
I would like to make my custom placeholder through my plugin

**Comment by LlmDl** [comment]
What did you want to put in your placeholder?

**Comment by ghost** [comment]
String, town ideology

пн, 11 апр. 2022 г. в 20:07, LlmDl ***@***.***>:

**Reply by LlmDl** [reply]
This is the guide for adding placeholders to your plugin: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Hook-into-PlaceholderAPI
